### ISSUE TITLE
[flux-core v0.13] example10/compute.lua: f:kvs_get() function no longer implemented

### ISSUE THREAD

**Issue by cmoussa1** [flux-framework/flux-workflow-examples#39]
Going through updating Example 10, and in **compute.lua**, the `kvs_get` function is used in the following fashion:

```
local f = assert (require 'flux' .new ())
.
.
.
-- conduit is the KVS key that conduit module puts to publishing
-- the rank in which it is loaded
cr = f:kvs_get ("conduit") or -1 

if cr == -1 then
    print ("conduit key is not available")
    os.exit (1)
end
.
.
.
```

This produces the following stack trace: 

```
bash-4.2$ flux mini submit -N 1 -n 1 ./compute.lua 1
6079744311296
bash-4.2$ flux job attach 6079744311296
flux-job: task(s) exited with exit code 1
lua: ./compute.lua:20: attempt to call method 'kvs_get' (a nil value)
stack traceback:
	./compute.lua:20: in main chunk
	[C]: ?
``` 

I'm assuming since the Lua bindings are deprecated, we should find another way to replicate this functionality if we want to keep this example with the new execution system.

**Comment by grondo** [comment]
Yeah, maybe a Python instead of Lua script here would be a much better example.

**Comment by dongahn** [comment]
I agree with @grondo.

**Comment by cmoussa1** [comment]
This issue was resolved with PR #44.
